tests/e2e/utils/workspace/TestWorkspaceUtil.ts

```diff
@@ -14,30 +14,41 @@ import { DriverHelper } from '../DriverHelper';
 import { CLASSES } from '../../inversify.types';
 import 'reflect-metadata';
 import * as rm from 'typed-rest-client/RestClient';
-import { error } from 'selenium-webdriver';
 import { WorkspaceStatus } from './WorkspaceStatus';
 import { ITestWorkspaceUtil } from './ITestWorkspaceUtil';
+import axios from 'axios';
+import querystring from 'querystring';
+enum RequestType {
```

Comment:
I guess it should be extracted to a dedicated file

Reply:
I don't expect this enum to be used in any other file, that's why I put it directly to the class. As this enum is really short, I would keep it here.